.github/workflows/push-2.6.yaml

```diff
@@ -28,3 +28,12 @@ jobs:
 
     - name: Push assets
       run: git push origin dev-v2.6
+
+    - name: Stage released/ directory
+      run: FROM_WORKFLOW=1 RELEASE_REPO_URL=https://github.com/rancher/charts.git RELEASE_BRANCH=release-v2.6 make stage-release
+
+    - name: Add released/ to branch
+      run: git add . && git -c user.name="actions" -c user.email="actions@github.com" commit -m "Regenerate released directory" -m "$(git log -b dev-v2.5-source --oneline -n1 --pretty=format:'%B')" || true
+
+    - name: Push assets
+      run: git push origin dev-v2.6
```

Comment:
Whats point of adding all this to 2.6 if we are going to make them fully separate branches ?

Reply:
At the current moment, they aren't separate, but when we separate them, we can drop this Github Workflow entirely

Reply:
We could repurpose this PR to drop the 2.6 workflows entirely in favor of manual forward-ports.

Reply:
Talked offline, we should keep this working for now until we confirm its being repurposed.